index.html

```diff
@@ -10,11 +10,13 @@
   </head>
   <body>
     <header>
-      <h1 id="logo">Aziz Mejri</h1>
-      <button type="button" class="menu-bars">
-        <i class="fas fa-bars"></i>
-      </button>
-      <nav id="desktop-nav">
+      <div id="mobile-nav">
+        <h1 id="logo">Aziz Mejri</h1>
+        <button type="button" class="menu-bars" id="switch-menu">
+          <i class="fas fa-bars"></i>
```

Comment:
- [ ] The menu bar should be responsible for opening the whole nav but that it closes it while the (X) opens it. That means it is working the opposite way. Kindly make the menu bar open it and make the (X) button close it.